### PR TITLE
Dit 1032 Quick  DAC  changes for Trader UI

### PR DESCRIPTION
**Pull request by pllaurado**

### CODE REVIEW

File: app/views/components/input_radio.scala.html [hunk 1]
```diff
@@ -18,28 +18,32 @@
...
 )(implicit messages: Messages)
 
 <div class="form-group @if(field.hasErrors){form-group-error}">
     <fieldset id="@{field.id}">
         <legend>
-            <div class="bold-small @if(legendClass.nonEmpty){@{legendClass.get}}">@legend</div>
+            @if(labelHtml.isDefined){
+               @labelHtml
+            }
+            @if(contentHtml.nonEmpty){
```
Comment (retroredge) [inline app/views/components/input_radio.scala.html]: I'd prefer to see `isDefined` here. `nonEmpty` make it look like a collection.

You use `isDefined` on line 31 so to swap here is confusion IMO :)

File: app/views/components/input_date.scala.html [hunk 2]
```diff
@@ -35,9 +35,9 @@
         <span class="form-label bold @if(legendClass.nonEmpty){@legendClass}">
           @label
         </span>
-        @if(hint.nonEmpty){
-            <span class="form-hint" id="@{id}-date-hint">@hint</span>
-        }
+            @if(hint.nonEmpty){
```
Comment (retroredge) [inline app/views/components/input_date.scala.html]: I'd prefer to see `isDefined` here. `nonEmpty` make it look like a collection.

File: app/views/components/input_text.scala.html [hunk 3]
```diff
@@ -16,16 +16,21 @@
...
         maxlength: Int = 1000
 )(implicit messages: Messages)
 
 <div class="form-group @if(field.hasErrors){form-group-error}">
     <label class="form-label" for="@{field.id}">
-        <div class="@if(labelClass.nonEmpty){@labelClass}">@label</div>
+        @labelHtml.map { lb => @lb }
+
+        @if(label.isDefined){
+            <div class="@if(labelClass.nonEmpty) {@labelClass}">@label</div>
```
Comment (retroredge) [inline app/views/components/input_text.scala.html]: I'd prefer to see `isDefined` here. `nonEmpty` make it look like a collection.

File: app/views/components/input_yes_no.scala.html [hunk 4]
```diff
@@ -16,28 +16,33 @@
...
+        noAssoc: Option[String] = None
 )(implicit messages: Messages)
 
 
 <div class="form-group @if(field.hasErrors){form-group-error}">
     <fieldset  id="@{field.id}">
-
         <legend>
-            <div @if(labelClass.nonEmpty){class="@labelClass"}>@label</div>
+            @if(label.nonEmpty){
```
Comment (retroredge) [inline app/views/components/input_yes_no.scala.html]: I'd prefer to see `isDefined` here. `nonEmpty` make it look like a collection.

File: app/views/components/input_yes_no.scala.html [hunk 5]
```diff
@@ -16,28 +16,33 @@
...
+                <div @if(labelClass.nonEmpty) {class="@labelClass"}>@label</div>
```
Comment (retroredge) [inline app/views/components/input_yes_no.scala.html]: The if condition is repeated here

Comment (pllaurado) [reply app/views/components/input_yes_no.scala.html]: not the same conditin

**Pull request merged**